jvb/src/main/java/org/jitsi/videobridge/AbstractEndpoint.java

```diff
@@ -93,6 +94,8 @@
 
     protected final EventEmitter<EventHandler> eventEmitter = new EventEmitter<>();
 
+    private VideoType videoType = VideoType.CAMERA;
```

Comment:
I have a vague memory of a 'start with screenshare' option in the client...if so, is it problematic to set this to camera by default?  Will the client update with screenshare right away in that case?

If that feature exists then ignore the comment :)

Reply:
Well, the client could start with any of them. My thinking was to use the most common case as a default, and assume that the client will send updates at least any time it uses a non-default.

@jallamsetty1 does that seem reasonable?

Reply:
The `startScreenSharing` option still exists but yes client can always send an update when the video source type is non-default.